save_scummer/backup.py

```diff
@@ -38,6 +39,14 @@ def get_included_files(source_pattern: StrOrPath) -> List[Tuple[Path, Path]]:
     return [(path, path.relative_to(base_dir)) for path in abs_paths if str(path) != base_dir]
 
 
+def timestamp_path_fragment(timestamp: datetime) -> str:
+    """Windows paths can't have colons, so preprocess the timestamp"""
+    timestamp_path_fragment = timestamp.isoformat()
+    if os.name == 'nt':
```

Comment:
I would be fine with changing the format for all OSs. So this could be:
```py
timestamp.isoformat().replace(':', '')
```

Or, if you prefer, a `strftime()` format like:
```py
timestamp.strftime('%y-%m-%dT%H-%M-%S')
```